ldregistry/config/language/messages/en.properties

```
@@ -63,7 +63,7 @@ checkUri.label=Check URI
 checkUri.heading=Check whether a URI is registered
 checkUri.button=Check
 checkUri.uri=URI to check
-checkUri.notFound=<strong>Not found.</strong> URI <{0}> does not seem to be registered.
+checkUri.notFound=<strong>Not found.</strong> =Sorry, the requested URI was not found on this site.
```

```suggestion
checkUri.notFound=<strong>Not found.</strong> Sorry, the requested URI was not found on this site.
```
An extra `=` had sneaked in there. Line 80 needs adjusting too.